test/functional/Java9andUp/src/org/openj9/test/java/lang/Test_Fma_Strict.java

```diff
@@ -0,0 +1,549 @@
+/*******************************************************************************
+ * Copyright (c) 2001, 2019 IBM Corp. and others
```

Comment:
Same comment as above for the early copyright date.

Reply:
Fixed in 1907302